src/main/java/hudson/plugins/testlink/result/AbstractTestNGResultSeeker.java

```diff
@@ -58,7 +57,7 @@
 	
 	public static final String TEXT_XML_CONTENT_TYPE = "text/xml";
 
-	protected final TestNGParser parser = new TestNGParser();
+	protected transient TestNGParser _parser = null;

```

Comment:
The class could be actually made static, but it needs update of the bundled lib

Reply:
Maybe should be `volatile`.

Reply:
Does not matter much. In the worst case there will be two instances of a same class which actually has no dynamic context at all. The class methods should be reworked to static, I just decided not to do breaking changes in this PR